spec/policies/comment_policy_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
```

Comment:
Not all tests are available here. Also, please don't forget to update tests after updating policy after PR review.